ultravox/model/ultravox_config.py

```diff
@@ -20,6 +21,20 @@ class LoraConfigSimplified:
     )
 
 
+class LossFunction(str, Enum):
+    CrossEntropy = "ce"
+    KL_Divergence = "kl"
+
+
+@dataclasses.dataclass
+class LossConfig:
+    loss_function: LossFunction = LossFunction.KL_Divergence
+    kl_temperature: float = 2.0
+
```

Comment:
suggest making this a `@property`

Reply:
Sounds like the right thing to do. Please suggest how to do it.

Reply:
This should be sufficient:

```suggestion
@property
```